docs/releases/4.2.txt

```diff
@@ -42,8 +42,14 @@ supports both libraries.
 Support for ``psycopg2`` is likely to be deprecated and removed at some point
 in the future.
 
+  .. admonition:: psycopg3 introduces breaking changes over psycopg2
+
+     Before upgrading to psycopg3 please be aware of any changes you may need to
+     make to raw queries as documented in: `Differences from psycopg2`_.
```

Comment:
We've avoided the `pyscopg3` terminology as the package is actually called `psycopg`

```suggestion
.. admonition:: ``psycopg >= 3`` introduces breaking changes over ``psycopg2``

   Before upgrading please be aware of any changes you may need to make to account for `differences from psycopg2`_
```